### ISSUE TITLE
when will double pass win at only 20+ moves

### ISSUE THREAD

**Issue by l1t1** [gcp/leela-zero#2142]
http://zero.sjeng.org/match-games/5c373143f06758029e306bfb

**Comment by Vandertic** [comment]
That client has something wrong

**Comment by Ishinoshita** [comment]
Series of first line or second line moves from opening on Elf side. Two games played by same client

**Comment by MartinDevelopment** [comment]
That is weird, is running two different network sizes breaking something for that client?

**Comment by l1t1** [comment]
the client must used a modified leelaz program

**Comment by l1t1** [comment]
only occur when test with elfv1 http://zero.sjeng.org/match-games/5c138ad7469ad81ed7b80260

**Comment by Ishinoshita** [comment]
Probably a different issue, if any (*theoretically*, that may happen with - n - m30), and so more for the anecdoct, a *self-play* game terminated by two pass moves, after 10 moves! 
http://zero.sjeng.org/view/188df66565a87c39ad9e2006ec23af593d7cb537b304782fcbb089a1bc872da1?viewer=wgo

*@Friday9, I beat you with high margin ;-) *

**Comment by l1t1** [comment]
slefplay game may radom pass @Ishinoshita

**Comment by Ishinoshita** [comment]
@l1t1 Agree in the principle.

In practice, even if pushed high by noise, pass move is pushed down in search by Q value in early phase of the game, unless the game is fully out of balance and win rate has plummeted to very low value. So, in very early opening, like at move 9 in the linked game, with no obvious blunder played by either side (all moves look reasonable to me), White picking pass at move 9 is quite surprising. Next Black pass also surprising for the same reason (pass move raw policy pretty close to 0, so noise again necessary + 'luck' with T=1). All in all, this two consecutive pass at move 10, with blunder before, are very very unlikely.

I have observed early two pass only at 60+ moves so far, and a blunder was involved and win rate had plummeted and the policy had already flattened, what helps two consecutive pass been picked. I'm talking of statistic at the scale of 3200 games batch, that I did 10's of times. Never observed so short self-play game before.

So, I just report as an anecdoct, as a 'clein d'œil' to @Friday9i who recently asked if two pass at move 70 was something normal ;-)!!!
